opal-ui/src/components/admin/providers/AddIdentityProviderDialog.vue

```diff
@@ -0,0 +1,348 @@
+            dense
+            type="text"
+            :label="$t('identity_provider.provider_url')"
+            :hint="$t('identity_provider.provider_url_hint')"
+            class="q-mb-md"
+            lazy-rules
+            :rules="[validateUri('validation.identity_provider.provider_url_format', false)]"
+          >
+          </q-input>
+          <q-input
+            v-model="newProvider.groups"
+            dense
+            type="text"
+            :label="$t('groups')"
+            :hint="$t('identity_provider.groups_hint')"
+            class="q-mb-md"
+            lazy-rules
+          >
+          </q-input>
+          <q-input
+            v-model="newProvider.scope"
+            dense
+            type="text"
+            :label="$t('identity_provider.scope')"
+            class="q-mb-md"
+            lazy-rules
+            :rules="[(val) => true]"
+          >
+            <!-- NOTE: to render hint in a with HTML the fields needs rules -->
+            <template v-slot:hint>
+              <div v-html="$t('identity_provider.scope_hint', { openid: '<code>openid</code>' })"></div>
+            </template>
+          </q-input>
+          <q-input
+            v-model="newProvider.usernameClaim"
+            dense
+            type="text"
+            :label="$t('identity_provider.username_claim')"
+            :hint="$t('identity_provider.username_claim_hint')"
+            class="q-mb-md"
+            lazy-rules
+          >
+          </q-input>
+          <q-select
+            v-model="groupsMapping"
+            :options="groupsMappingOptions"
+            dense
+            :label="$t('laguages')"
+            class="q-mb-md q-pt-md"
+          />
+          <q-input
+            v-if="isGroupsMappingByClaim"
+            v-model="newProvider.groupsClaim"
+            dense
+            type="text"
+            :placeholder="$t('groups')"
+            :hint="$t('identity_provider.groups_claim_hint')"
+            class="q-mb-md"
+            lazy-rules
+          >
+          </q-input>
+          <q-input
+            v-else
+            v-model="newProvider.groupsScript"
+            dense
+            rows="6"
+            type="textarea"
+            :placeholder="groupsJavascriptPlaceholder"
+            :hint="$t('identity_provider.groups_javascript_hint')"
+            class="q-mb-md"
+            lazy-rules
+          >
+          </q-input>
+
+          <div class="q-py-md">
+            <q-list>
+              <q-expansion-item
+                dense
+                switch-toggle-side
+                header-class="text-primary text-caption q-pl-none"
+                :label="$t('advanced_options')"
+              >
+                <q-checkbox dense v-model="newProvider.useNonce" class="q-my-md">
+                  <template v-slot:default>
+                    <span
+                      class="text-secondary"
+                      v-html="$t('identity_provider.use_nonce', { url: useNonceDefinition })"
```

Comment:
Url link does not work, check box captures the event. You do not need this fancy link (or put it in a `<div class="text-hint">` below  the q-checkbox

![image](https://github.com/obiba/opal/assets/2581149/1db727bc-bb68-4e7a-a45a-f40aeb544a9a)

Reply:
fixed